### DISCUSSION TITLE
Small window for buffer info pops up on session restore

### DISCUSSION THREAD

**Discussion by alankritjoshi** [olimorris/persisted.nvim#131] [Q&A] [answered]
I don't see this when I run open a specific file or if there's no session to restore. Only happens when restore is triggered.

Not sure how to find out what is triggering this buffer to show up. Any ideas on what's happening or a way to debug the cause?

<img width="1332" alt="Screenshot 2024-05-17 at 11 00 24 AM" src="https://github.com/olimorris/persisted.nvim/assets/13654555/ef21e9cb-8996-4bac-aa9e-08601474b412">

**Comment by olimorris** [comment, marked as answer]
When you're closing Neovim, this must be getting written into the session. Take a look at the plugin's events that you can hook into. I'd advise removing certain buffers prior to saving the session.
